projects/plugins/jetpack/tests/e2e/lib/pages/wp-admin/recommendations.js

```diff
@@ -16,19 +16,19 @@ export default class RecommendationsPage extends WpPage {
 	}
 
 	get siteTypePersonalCheckboxSel() {
-		return '.jp-checkbox-answer__container:nth-child(1) input';
+		return '.jp-checkbox-answer__container input#site-type-personal';
 	}
 
 	get siteTypeBusinessCheckboxSel() {
-		return '.jp-checkbox-answer__container:nth-child(2) input';
+		return '.jp-checkbox-answer__container input#site-type-business';
 	}
 
-	get siteTypeOtherCheckboxSel() {
-		return '.jp-checkbox-answer__container:nth-child(4) input';
+	get siteTypeStoreCheckboxSel() {
+		return '.jp-checkbox-answer__container input#site-type-store';
 	}
 
-	get siteTypeStoreCheckboxSel() {
-		return '.jp-checkbox-answer__container:nth-child(3) input';
+	get siteTypeOtherCheckboxSel() {
+		return '.jp-checkbox-answer__container input#site-type-other';
 	}
```

Comment:
Depending on the order of the element introduces brittleness. The element ID guarantees that tests won't break if we change the order of the questions.